src/platform/cannonlake/power_down.S

```diff
@@ -144,6 +144,21 @@ beqz pu32_hpsram_mask, _PD_SLEEP
 	m_cavs_set_hpldo_state temp_reg0, temp_reg1, temp_reg2
 
 //TODO: add sending IPC reply from L1$ locked code
+_PD_SEND_IPC:
+/* Send IPC to host informing of PD completion - Clear BUSY
+ * bit by writing IPC_DIPCTDR_BUSY to IPC_DIPCTDR
+ * and writing IPC_DIPCTDA_BUSY to IPC_DIPCTDA
+ */
+	movi temp_reg0, IPC_HOST_BASE
+	l32i temp_reg1, temp_reg0, IPC_DIPCTDR
```

Comment:
@RanderWang just to confirm this was based on APL version and modified for differences in CNL IPC registers ?

Reply:
yes, It is based on APL version and I change the registers for CNL